### PR TITLE
Build prepare client release

### PR DESCRIPTION
**Pull request by gmastrapas**

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -24,28 +52,44 @@ jobs:
         env:
           release_token: ${{ secrets.FINETUNER_RELEASE_TOKEN }}
 
-  create-release:
-    name: Create release
-    needs: update-doc
+#  create-release:
+#    name: Create GitHub release
+#    needs: update-docs
```
Comment (jupyterjazz) [inline .github/workflows/cd.yml:57]: Why is this commented?

Comment (gmastrapas) [reply .github/workflows/cd.yml:57]: it needs additional stuff to work. I left it commented out for now

File: setup.py [hunk 2]
```diff
@@ -1,64 +1,96 @@
...
+from setuptools import find_packages, setup
 
-if sys.version_info < (3, 7, 0):
-    raise OSError(f'Finetuner requires Python >=3.7, but yours is {sys.version}')
+# package name
+_name = 'finetuner-client'
 
+
+# package version
+__version__ = '0.0.0'
```
[the remark this answers is not in the document]
Comment (gmastrapas) [reply setup.py:10]: the version is defined in `finetuner/__init__.py`. This is just a fallback if the file is not found

**Review by jupyterjazz — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**